CHANGELOG.md

```diff
@@ -235,6 +235,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Fixed issue where the CLI could not pass a `Profiler` to the `Trainer` ([#13084](https://github.com/PyTorchLightning/pytorch-lightning/pull/13084))
 
 
+- Fixed logging on step level for eval mode ([#12184](https://github.com/PyTorchLightning/pytorch-lightning/pull/12184))
```

Comment:
This is too general. You need to mention that this is fixed for the case of multiple dataloaders. As I understand, there is no known bug for single dataloader use cases.

Reply:
What about "Fixed logging's step values when multiple dataloaders are used during evaluation"?

I'll resolve this comment and change it in a separate PR as it's ready to go otherwise. Save the trees :evergreen_tree: